docker/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+# To rebuild and run this container:
+# docker compose build && docker compose up -d && docker logs -f pudl_etl
+
+services:
+  pudl-etl:
+    environment:
+      - API_KEY_EIA
+    env_file:
+      - .env
+    build:
+      context: ../
+      dockerfile: docker/Dockerfile
+    image: catalystcoop/pudl-etl:hello-docker
```

Comment:
Should figure out some way to use the current ref for the code that's checked out, or the software version, rather than `hello-docker`